awx/main/tasks/jobs.py

```diff
@@ -82,6 +82,190 @@
 logger = logging.getLogger('awx.main.tasks.jobs')
 
 
+class RunnerCallback:
+    def __init__(self, BaseTaskObject):
+        self.event_data_key = BaseTaskObject.event_data_key
+        self.parent_workflow_job_id = BaseTaskObject.parent_workflow_job_id
+        self.job_created = BaseTaskObject.job_created
+        self.host_map = BaseTaskObject.host_map
+        self.guid = BaseTaskObject.guid
+        self.recent_event_timings = BaseTaskObject.recent_event_timings
+        self.instance = BaseTaskObject.instance
```

Comment:
style-wise, `BaseTaskObject` looks like it should be a class, I'd prefer a style like `task_object` so it's more obviously an instantiated object.